addons/amxmodx/scripting/rt_bonus.sma

```diff
@@ -137,7 +134,9 @@ public RegisterCvars()
 		FCVAR_NONE,
 		"0 - do not issue armor, 1 - bulletproof vest, 2 - bulletproof vest with helmet",
 		true,
-		0.0),
+		0.0,
+        true,
+        2.0),
```

Comment:
It is necessary to remove from the PR

Reply:
enum ArmorType
{
	ARMOR_NONE,     // 0 No armor
	ARMOR_KEVLAR,   // 1 Body vest only
	ARMOR_VESTHELM, // 2 Vest and helmet
};